scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -38,7 +38,7 @@ def call(final pipelineContext) {
   // for Python, smoke only oldest and latest supported versions
   def SMOKE_STAGES = [
     [
-      stageName: 'Py3.6 Smoke', target: 'test-py-smoke', pythonVersion: '3.6',timeoutValue: 8,
+      stageName: 'Py3.7 Smoke', target: 'test-py-smoke', pythonVersion: '3.7',timeoutValue: 8,
```

Comment:
why we need this? haven't you experienced a problem with urllib3 on python 3.7?

Reply:
Good one, don't remember. I think that xgboost 1.6 cannot be installed, but I handle that differently. I am going to run it again and will go back with the result :)

Reply:
but it run successfully on my branch

Reply:
Py3.6 is working, I am going to revert this change